firmware/projects/TMS/platforms/stm32f767/CMakeLists.txt

```diff
@@ -3,5 +3,5 @@
 
 add_subdirectory(cubemx)
 
-target_sources(bindings PUBLIC bindings.cc)
+target_sources(bindings PUBLIC bindings.cc os.cc)
```

Comment:
We weren't compiling the os.cc file in mainline??

Reply:
i have no idea how it was working. I could've sworn we created an `os` CMake library (analogous to the `bindings` library), but I don't see it in TMS or DemoProjectRTOS. I'll have to find where that went